satnogsclient/scheduler/tasks.py

```diff
@@ -234,84 +253,99 @@ def ecss_listener(port, queue):
 
 def status_listener():
     logger.info('Started upsat status listener')
+    logger.info('Starting scheduler...')
+    scheduler.start()
+    interval = settings.NETWORK_API_QUERY_INTERVAL
+    scheduler.add_job(get_jobs, 'interval', minutes=interval)
+    msg = 'Registering `get_jobs` periodic task ({0} min. interval)'.format(interval)
+    print msg
+    interval = settings.NETWORK_API_POST_INTERVAL
+    msg = 'Registering `post_data` periodic task ({0} min. interval)'.format(interval)
+    print msg
+    scheduler.add_job(post_data, 'interval', minutes=interval)
+    print 'Scheduler jobs are ', scheduler.print_jobs()
+    tf = Process(target=task_feeder, args=(settings.TASK_FEEDER_TCP_PORT, settings.TASK_LISTENER_TCP_PORT,))
+    tf.start()
+    os.environ['TASK_FEEDER_PID'] = str(tf.pid)
     sock = Udpsocket(('127.0.0.1', settings.STATUS_LISTENER_PORT))
+    os.environ['BACKEND_TX_PID'] = '0'
+    os.environ['BACKEND_RX_PID'] = '0'
+    os.environ['BACKEND'] = ""
```

Comment:
nit: I suggest you always use single-quotes except in cases you really need double-quotes.